index.html

```diff
@@ -43,6 +43,153 @@ <h3>LET'S CONNECT</h3>
+                    <div class="external_link">
+                        <a href="#">see project</a>
+                    </div>
+                </footer>
+            </div>
+        </div>
+        <div class="project">
+            <img class="project_image" src="assets/Snapshoot Portfolio (1).png" alt="">
+            <div class="general">
+                <h3>Multi-post Stories</h3>
+                <div class="general_inf">
+                    <p>Canopy.Back-end.2023</p>
+                </div>
+                <p class="project_description">A daily selection of privately personalized reads; no 
+                    accounts or sign-ups required.</p>
+                <footer>
+                    <ul class="skills">
+                        <li>Html</li>
+                        <li>Css</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <div class="external_link">
+                        <a href="#">see project</a>
+                    </div>
+                </footer>
+            </div>
+        </div>
+        <div class="project">
+            <img class="project_image" src="assets/Snapshoot Portfolio (2).png" alt="">
+            <div class="general">
+                <h3>Tonic</h3>
+                <div class="general_inf">
+                    <p>Canopy.Back-end.2023</p>
+                </div>
+                <p class="project_description">A daily selection of privately personalized reads; no 
+                    accounts or sign-ups required.</p>
+                <footer>
+                    <ul class="skills">
+                        <li>Html</li>
+                        <li>Css</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <div class="external_link">
+                        <a href="#">see project</a>
+                    </div>
+                </footer>
+            </div>
+        </div>
+        <div class="project">
+            <img class="project_image" src="assets/Snapshoot Portfolio (3).png" alt="">
+            <div class="general">
+                <h3>Multi-post Stories</h3>
+                <div class="general_inf">
+                    <p>Canopy.Back-end.2023</p>
+                </div>
+                <p class="project_description">A daily selection of privately personalized reads; no 
+                    accounts or sign-ups required.</p>
+                <footer>
+                    <ul class="skills">
+                        <li>Html</li>
+                        <li>Css</li>
+                        <li>Javascript</li>
+                    </ul>
+                    <div class="external_link">
+                        <a href="#">see project</a>
+                    </div>
+                </footer>
+            </div>
+        </div>
+        
+
+    </div>
+    <section class="last-sect">
+        <section id="about-myself">
+        <div class="myself-abt">
+            <h1>About<br>Myself</h1>
```

Comment:
Kindly update this element to use another `<hx>` tag apart from `<h1>`🙏.  It is considered best practice for a webpage to have only one `<h1>` tag.

Reply:
I added an H1 tag for the about and myself elements.